landing-page/components/sections/talks/EpisodeSection.tsx

```diff
@@ -4,6 +4,22 @@ import Link from 'next/link'
 
 import { LoadingAnimation } from '@helpwave/common/components/LoadingAnimation'
 import { useQuery } from '@tanstack/react-query'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+
+type EpisodeTranslation = {
```

Comment:
```suggestion
type EpisodeSectionTranslation = {
```
Check also the other mentions of `Episode...`